autoscheduler/frontend/src/utils/html2canvas.js

```diff
@@ -0,0 +1,11 @@
+import * as html2canvas from 'html2canvas'; // This import is incorrect, so we made this a JS file
```

Comment:
Allegedly html2canvas already has type definitions, are they incorrect? I also noticed `package.json` uses v1.0.0 whereas the package is at v1.3.2, what's the reason for using an old version?

Reply:
We actually use `v1.0.0-rc.7` which was the most up-to-date version when I made this 3 months ago - they only started recently updating it again apparently, but I will update it to use the most up-to-date version.

And yes, the type definitions were incorrect. Seems like per https://github.com/niklasvh/html2canvas/issues/2628 that if we updated to TypeScript 3.8 (currently on 3.6.4) the types would work correctly, which I will try to do

Reply:
Ok so updated both and the issue is still occurring. I'm going to update `html2canvas` to `1.3.2` anyways but the typescript stuff is still messed up so the weird layout is going to remain the same.